cmd/entrypoint/main.go

```diff
@@ -41,6 +41,7 @@ var (
 	terminationPath     = flag.String("termination_path", "/tekton/termination", "If specified, file to write upon termination")
 	results             = flag.String("results", "", "If specified, list of file names that might contain task results")
 	waitPollingInterval = time.Second
+	timeout             = flag.String("timeout", "", "If specified, sets timeout for step")
```

Comment:
So there's https://golang.org/pkg/flag/#Duration which handles parsing for us, and will let us abstract away another parse...